_posts/newsletters/2022-01-31-newsletter.md

```diff
@@ -0,0 +1,205 @@
+
+<a name="workshop"></a>
+# **US-RSE Community Workshop**
+
+We are excited to announce that the first US-RSE Association [community building workshop](https://us-rse.org/first-community-workshop/) will be held April 26-27, 2022 in Princeton, NJ, after two years of delays due to COVID-19! This will be a workshop focused on planning the best path forward to grow the US-RSE Association.
+
+<a name="community-calls"></a>
+# **US-RSE Community Calls**
+
+The February community call will be on February 10, 2022 at 12 ET. We will be talking about Testing Frameworks and Processes.
+
+Most developers will agree that testing is an important part of software development. But is testing of research software any different than testing of other types of software? Is testing in a research context different from testing in general software enterprises? And how and what do you test in the first place? What can we learn from each other and from practices elsewhere?
+
+ The call will discuss, and share knowledge and experiences, around questions such as why you would want to test and what the benefits are, what technologies are out there, and what processes RSEs do (and don’t) employ to test their software. No matter if you have never heard of testing or if you are an expert, come join us for an interesting discussion and to hear and share some "testimonies" of what has worked and not worked for others.
+
+We have also changed the way we organize community calls. We meet two weeks prior to each community call to decide on a topic and discuss how to structure the call. Anyone is welcome to join the preparation call on [Zoom](https://mit.zoom.us/j/93471607274?pwd=aXUzNzZsWElpWnc4N2NXaWxOcVNlQT09).
+
+<a name="podcast"></a>
+# **RSE Stories Podcast**
+
+The [Research Software Engineer Stories podcast](http://us-rse.org/rse-stories/) has two new episodes posted. Check them out below!
+
+RSE Stories is looking for a new co-host and other volunteers to help with production. You don’t need to commit to anything: start with one episode and go from there. Interested? Reach out to Vanessa via Slack.
+
+* Chase Million: [No Shortage of Ideas](http://us-rse.org/rse-stories/2022/chase-million/)
+* Tim Haines: [For the Love of Learning](http://us-rse.org/rse-stories/2021/tim-haines/)
+
+<a name="scipy"></a>
+# **US-RSE at SciPy 2022**
+
+US-RSE members plan to submit a proposal for a [Birds of a Feather session at SciPy 2022](https://www.scipy2022.scipy.org/bof-sessions) about US-RSE. This will be a moderated panel discussion. If you are planning to attend SciPy 2022 in July in Austin and interested in representing US-RSE on the panel, please fill out this form: [https://forms.gle/Mx4qW1zH518Ev2Cm6](https://forms.gle/Mx4qW1zH518Ev2Cm6)
```

Comment:
Might want to clarify if "attend scipy" means in person or virtual or either?

Reply:
Fair question, but I don't know, and I'm not sure I'll get an answer tonight. I think since it's Austin, the assumption is in person